### PR TITLE
non-invasive change: range() by xrange() for python 2 loops

### PR DESCRIPTION
**Pull request by govarguz**
Changed range() by xrange() for all python 2.x loops

### CODE REVIEW

**Comment by MrTheodor** [comment]
LGTM; 
btw, there is a restriction in CPython for xrange argument to be in range of C long type. somthing like 10^19 ;)

File: src/MultiSystem.py [hunk 1]
```diff
@@ -19,63 +19,18 @@
 #  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
 
 
-r"""
-**************************
+"""
+************************
 **espressopp.MultiSystem**
-**************************
-
```
Comment (MrTheodor) [inline src/MultiSystem.py]: Why do you remove whole documentation section?

File: src/ParallelTempering.py [hunk 2]
```diff
@@ -19,70 +19,11 @@
 #  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
 
 
-r"""
-********************************
+"""
+******************************
 **espressopp.ParallelTempering**
-********************************
-
```
Comment (MrTheodor) [inline src/ParallelTempering.py]: here also?

File: src/ParallelTempering.py [hunk 3]
```diff
@@ -97,7 +38,7 @@ def __init__(self, NumberOfSystems = 4, RNG = None):
         print "+ Class is not available in the current version. This Class will be back soon.  +"
         print "+ Multisystem simulations are still possible but have to be setup manually.     +"
         print "+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
-        return 0 
+        #return 0
```
Comment (MrTheodor) [inline src/ParallelTempering.py]: if this really works then please remove those prints above

Comment (govarguz) [inline src/ParallelTempering.py]: sure!

**Comment by acfogarty** [comment]
Was it really necessary to have so many separate commits with identical commit messages? I suggest we squash them into one when merging, for a more readable git log.

**Comment by govarguz** [comment]
ohh, very True! however they cannot be "squashed they are ate least 2 different functionalities, @MrTheodor is aware of.

**Comment by acfogarty** [comment]
You can just squash all the copyright commits together, e.g. as explained here:

http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/5201642#5201642

**Comment by acfogarty** [comment]
I also think it's a pity to unnecessarily make Travis waste resources going through so many builds. You could kill all the builds except the last one.

**Comment by govarguz** [comment]
cool!

**Comment by MrTheodor** [comment]
Could you rebase with the master:

This branch is out-of-date with the base branch
Merge the latest changes from master into this branch.

**Comment by junghans** [comment]
And @govarguz you can push all the trivial commits at once, which will run the CI only on that last one.
Even if Travis is free, we don't want to waste it ;-)

**Pull request closed without merging**